packages/electron-updater/src/NsisUpdater.ts

```diff
@@ -109,6 +109,11 @@ export class NsisUpdater extends BaseUpdater {
       args.push("--force-run")
     }
 
+    if (this.installDir && typeof this.installDir === 'string') {
```

Comment:
I think we shouldn't need to check for type = string here since that should be autovalidated by the schema validator

Reply:
ok